swagger.yaml

```diff
@@ -540,6 +592,43 @@ components:
       bearerFormat: JWT
 
   schemas:
+    User:
+      type: object
+      description: User Data
+      required:
+        - id
+        - firstName
+        - lastName
+        - isActive
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: User's ID in Keycloak.
+        firstName:
+          type: string
+          description: User's first name.
+        lastName:
+          type: string
+          description: User's last name.
+        email:
+          type: string
+          description: User's email.
+        isActive:
+          type: boolean
+          description: Defining is user is active e.g. is employed.
+        passedQuestionnaires:
+          type: array
+          items:
+            type: integer
```

Comment:
An array of integers? Can you explain how this works, I would maybe have expected an array of strings/ questionnaire Ids or perhaps an array of objects with the Q id and the user score?

Reply:
The integer represents the questionnaire id the user has passed, as the id is in integer format.